SU2_CFD/src/solvers/CEulerSolver.cpp

```diff
@@ -5764,9 +5860,189 @@ void CEulerSolver::SetActDisk_BCThrust(CGeometry *geometry, CSolver **solver_con
+                        per unit area (Fr and Ft) are equal to zero, while the axial force per unit area (Fa) is computed using
+                        a linear interpolation in order to avoid a mathematical singularity at actuator disk center. ---*/
+                  if (rad_v[0] == 0.0){
+                      Fa[0] = (((2*Dens_FreeStream*pow(Vel_FreeStream[0],2))/
+                              (pow(AD_J,2)*PI_NUMBER))*((dCt_v[1] - dCt_v[0])/rad_v[1])) / config->GetPressure_Ref();
+                      Ft[0] = 0.0;
+                      Fr[0] = 0.0;
+                  }
+                  else{
+                    Fa[0] = (dCt_v[0]*(2*Dens_FreeStream*pow(Vel_FreeStream[0],2))/
+                            (pow(AD_J,2)*PI_NUMBER*rad_v[0])) / config->GetPressure_Ref();
+                    Ft[0] = (dCp_v[0]*(2*Dens_FreeStream*pow(Vel_FreeStream[0],2))/
+                            ((AD_J*PI_NUMBER*rad_v[0])*(AD_J*PI_NUMBER*rad_v[0]))) / config->GetPressure_Ref();
+                    Fr[0] = (dCr_v[0]*(2*Dens_FreeStream*pow(Vel_FreeStream[0],2))/
+                            (pow(AD_J,2)*PI_NUMBER*rad_v[0])) / config->GetPressure_Ref();
+                  }
+
+                  /*--- Loop over the radial stations. Computation of Fa (axial force per unit area), Ft (tangential force per unit area)
+                        and Fr (radial force per unit area).
+                        These equations are not valid if the freestream velocity is equal to zero (hovering condition not enabled yet). ---*/
+                  for (iEl = 1; iEl < nRow; iEl++){
+                    Fa[iEl] = (dCt_v[iEl]*(2*Dens_FreeStream*pow(Vel_FreeStream[0],2))/
+                              (pow(AD_J,2)*PI_NUMBER*rad_v[iEl])) / config->GetPressure_Ref();
+                    Ft[iEl] = (dCp_v[iEl]*(2*Dens_FreeStream*pow(Vel_FreeStream[0],2))/
+                              ((AD_J*PI_NUMBER*rad_v[iEl])*(AD_J*PI_NUMBER*rad_v[iEl]))) / config->GetPressure_Ref();
+                    Fr[iEl] = (dCr_v[iEl]*(2*Dens_FreeStream*pow(Vel_FreeStream[0],2))/
+                              (pow(AD_J,2)*PI_NUMBER*rad_v[iEl])) / config->GetPressure_Ref();
+                  }
+
+                  /*--- Loop over the marker nodes. ---*/
+                  for (iVertex = 0; iVertex < geometry->nVertex[iMarker]; iVertex++) {
+                     /*--- Get the coordinates of the current node. ---*/
+                     iPoint = geometry->vertex[iMarker][iVertex]->GetNode();
+                     P = geometry->nodes->GetCoord(iPoint);
+
+                     /*--- Computation of the radius coordinates for the current node. ---*/
+                     GeometryToolbox::Distance(nDim, P, AD_Center, r);
+
+                     /*--- Computation of the non-dimensional radius for the current node. ---*/
+                     r_ = GeometryToolbox::Distance(nDim, P, AD_Center) / AD_Radius;
```

Comment:
You could use just `Norm` on the `r` computed above.

Reply:
Did it.